cluster_management/src/main/java/com/pinterest/rocksplicator/Participant.java

```diff
@@ -187,13 +191,20 @@ public static void disconnectHelixManager() throws Exception {
     }
   }
 
+  public HelixManager getHelixManager() {
+    return helixManager;
+  }
+
 
-  private Participant(String zkConnectString, String clusterName, String instanceName,
-                     String stateModelType, int port, String postUrl,
-                     boolean useS3Backup, String s3BucketName, boolean runSpectator) throws Exception {
+  public Participant(String zkConnectString, String clusterName, String instanceName,
+                      String stateModelType, int port, String postUrl,
+                      boolean useS3Backup, String s3BucketName, boolean runSpectator)
+      throws Exception {
     helixManager = HelixManagerFactory.getZKHelixManager(clusterName, instanceName,
         InstanceType.PARTICIPANT, zkConnectString);
 
+    StateMachineEngine stateMach = helixManager.getStateMachineEngine();
```

Comment:
If we do the suggested changes below, we should move this to be after the following if else.